fpo-web/src/app/survey/primary.component.html

```diff
@@ -9,10 +9,15 @@
         [initialMode]="initialMode"
       ></app-survey-view>
     </div>
-    <div class="col-md-12" *ngIf="submitted">
+    <div class="col-md-12 mt-4" *ngIf="submitted">
       <p><b>Thank-you for choosing how you wish to attend your traffic hearing.</b></p>
-    
-      <p>A copy of your completed form is attached for your records.</p>
+       
+      <div id="emailSent" *ngIf="emailStatus; else emailFailed">
+        <p>A copy of your completed form has been sent to you by email.</p>
+      </div>
+      <ng-template #emailFailed>
+        <p>Sorry, we were unable to send you a copy of your completed form by email.  Please save your completed form by clicking at "View Your Document".</p>
```

Comment:
This message should be highlighted to make it obvious there was an issue and action needs to be taken.  Perhaps larger bold text.

Reply:
I made the changes to the email failed text. I made it bold and little larger (h4) than other text. I also tried making it more larger with <h3>but it didn't look good. Let me know if you think making it different color make it obvious.